README.md

```diff
@@ -46,10 +46,32 @@ We decided to apply _English_ as the primary project language.
 
 Consequently, all content will be made available primarily in English. We also ask all interested people to use English as language to create issues, in their code (comments, documentation etc.) and when you send requests to us. The APIs itself and all end-user facing content will be made available in other languages as needed.
 
-## Meetings
-* Meetings are held virtually
-* Schedule: Each second Thursday in a month, 4pm CET
-* Meeting link: https://dtag.webex.com/dtag/j.php?MTID=m79544089ecb17cfa5046b7e1982926f3
+## Steering Committee
+The Steering Committee is the oversight body for the CAMARA Project. The Steering Committee currently consists of the following participants:
+| Partner                     | Name(s)                                                |
+| --------------------------- | ------------------------------------------------------ |
+| 5GFF	| Janine Grasso (Miguel Carames, Kevin Smith) |
+| AT&T	| Wiley Wilkins (Barry Elia) |
+| Capgemini	| Shamik Mishra |
+| Deutsche Telekom AG	| Nathan Rader (Markus Kümmerle) |
+| Ericsson	| Bart van Kaathoven (Jan Friman) |
+| Google Cloud	| Sridhar Gollapudi |
+| GSMA	| Henry Calvert |
+| IBM	| Jason Hunt (Zyg Lozinski) |
+| Intel	| Petar Torre |
+| Kandy	| Philip Lintell (Ralph Page) |
+| KDDI	| Toshiyasu Wakayama |
```

Comment:
Isn't Meta part of the SC?

Reply:
No, Meta is still in progress to give the ok to show their logo